CHANGELOG.md

```diff
@@ -12,13 +12,16 @@ All notable changes to this project will be documented in this file.
 - [BREAKING]: Removed tools image (reporting task job and init container) and replaced with NiFi product image. This means the latest stackable version has to be used in the product image selection ([#397])
 - Fixed the RoleGroup `selector`. It was not used before. ([#401])
 - Refactoring of authentication handling ([#408])
+- [BREAKING]: Renamed global `config` to `clusterConfig` ([#417])
+- [BREAKING]: Moved `zookeeper_configmap_name` to `clusterConfig` ([#417])
```

Comment:
The release 23.1 must be added and these entries must be moved to the new Unreleased section:

```md
## [Unreleased]

### Changed

- [BREAKING]: Renamed global `config` to `clusterConfig` ([#417])
- [BREAKING]: Moved `zookeeper_configmap_name` to `clusterConfig` ([#417])

[#417]: https://github.com/stackabletech/nifi-operator/pull/417

## [23.1.0] - 2023-01-23

### Changed

- Updated operator-rs to 0.31.0 ([#382], [#401], [#408])
- Do not run init container as root anymore and avoid chmod and chown ([#390])
- [BREAKING] Use Product image selection instead of version. `spec.version` has been replaced by `spec.image` ([#394])
- [BREAKING]: Removed tools image (reporting task job and init container) and replaced with NiFi product image. This means the latest stackable version has to be used in the product image selection ([#397])
- Fixed the RoleGroup `selector`. It was not used before. ([#401])
- Refactoring of authentication handling ([#408])

[#382]: https://github.com/stackabletech/nifi-operator/pull/382
[#390]: https://github.com/stackabletech/nifi-operator/pull/390
[#394]: https://github.com/stackabletech/nifi-operator/pull/394
[#397]: https://github.com/stackabletech/nifi-operator/pull/397
[#401]: https://github.com/stackabletech/nifi-operator/pull/401
[#408]: https://github.com/stackabletech/nifi-operator/pull/408
```

Reply:
https://github.com/stackabletech/nifi-operator/pull/417/commits/03d95773618486f8b54200dfa6f058c8598914b8